<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>243</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="title">
    <string>Regressor setup</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>N components</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Delays</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="delay_txt"/>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="nComponents_spin">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="nPulses_spin">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>99</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>N pulses</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="setRegressor">
        <property name="text">
         <string>Set regressor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="basisfile">
        <property name="text">
         <string>Basis file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="basisfileEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="newrefs">
        <property name="text">
         <string>Save new references</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="GraphicsLayoutWidget" name="basisCanvas">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
